lib/airbrake_ex/notifier.ex

```diff
@@ -35,16 +35,32 @@ defmodule AirbrakeEx.Notifier do
     end
   end
 
-  defp filter_parameters(params, filtered_keys) do
+  defp filter_parameters(params, filter_config) do
     for {key, value} <- params, into: %{} do
-      if Enum.member?(filtered_keys, to_string(key)) do
+      if ignore_parameter?(key, filter_config) or contains_password?(key) do
         {key, "***"}
       else
-        {key, value}
+        if is_map(value) do
+          {key, filter_parameters(value, filter_config)}
```

Comment:
Je pense que c'est safe to say qu'on veut toujours que ca vérifie recursivement, tant pour le password que le `ignore_parameter`